templates/download/up-squared-iot-grove-server.html

```diff
@@ -1,94 +1,77 @@
+  <div class="row--50-50 p-section">
+    <div class="col">
       <h1>Install Ubuntu Server on an UP<sup>2</sup> IoT Grove Development Kit</h1>
     </div>
-  </div>
-  <div class="row">
-    <div class="col-12 p-card">
-      <div class="u-equal-height row p-divider">
-        <div class="col-6 p-divider__block">
-          <h2>Install Ubuntu Server</h2>
-          <p>We will walk you through the steps of installing Ubuntu Server on the UP<sup>2</sup> IoT Grove Development Kit. At the end of this process, you will have a fully fledged development or production environment.</p>
-        </div>
-        <div class="col-6 p-divider__block">
-          <h3>Minimum requirements</h3>
-          <ul class="p-list">
-            <li class="p-list__item is-ticked">An UP2 IoT Grove Development Kit</li>
-            <li class="p-list__item is-ticked">A USB flash drive</li>
-            <li class="p-list__item is-ticked">An Ubuntu Server image</li>
-          </ul>
-        </div>
-      </div>
+    <div class="col">
+      <p>We will walk you through the steps of installing Ubuntu Server on the UP<sup>2</sup> IoT Grove Development Kit. At the end of this process, you will have a fully fledged development or production environment.</p>
+      <hr class="p-rule" />
+      <p class="p-heading--5">Minimum requirements</p>
+      <ul class="p-list--divided">
+        <li class="p-list__item is-ticked">An UP<sup>2</sup> IoT Grove Development Kit</li>
+        <li class="p-list__item is-ticked">A USB flash drive</li>
+        <li class="p-list__item is-ticked">An Ubuntu Server image</li>
+      </ul>
     </div>
   </div>
 </section>
 
-<section class="p-strip is-deep is-bordered">
-  <div class="u-fixed-width">
-    <h2>Installation instructions</h2>
+<section class="p-section">
+  <div class="row">
     <ol class="p-stepped-list--detailed">
       <li class="p-stepped-list__item">
-        <h3 class="p-stepped-list__title">
-          
+        <hr class="p-rule u-hide--large" />
+        <h2 class="p-stepped-list__title">
           Download Ubuntu Server
-        </h3>
+        </h2>
         <div class="p-stepped-list__content">
           <p>Get the correct Ubuntu Server image for your board:</p>
-          <ul class="u-no-margin--left">
-            <li>Intel provides a modified Ubuntu Server 16.04.3 LTS image shipped with a custom 4.10 kernel and the Intel IoT Developer Kit.</li>
-            <li>Download and copy the <a href="https://downloads.up-community.org/download/up-squared-iot-grove-development-kit-ubuntu-16-04-server-image/">Ubuntu Server 16.04 LTS</a> image on a USB flash drive by following the live USB Ubuntu Desktop tutorial for <a href="/tutorials/tutorial-create-a-usb-stick-on-ubuntu">Ubuntu</a>, <a href="/tutorials/tutorial-create-a-usb-stick-on-windows">Windows</a>, or <a href="/tutorials/tutorial-create-a-usb-stick-on-macos">macOS</a></li>
+          <ul class="p-list--divided u-no-margin--left">
```

Comment:
```suggestion
          <ul class="p-list--divided">
```
This isn't needed, Vanilla applies margin-left: 0 out of the box to divided lists
![image](https://github.com/canonical/ubuntu.com/assets/17607612/3bf88729-195e-4eb1-b416-516b9ee48a33)